docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -352,6 +352,27 @@ as described in [how to run the Flink image](#how-to-run-flink-image).
     # e.g. to distribute the custom image to your cluster
     docker push custom_flink_image
     ```
+  
+### Enabling Python
+
+To build a custom image which has Python and Pyflink prepared, you can refer to the following Dockerfile:
+{% highlight Dockerfile %}
+FROM flink
+
+# install python3 and pip3
+RUN apt-get update -y && \
+apt-get install -y python3.7 python3-pip python3.7-dev && rm -rf /var/lib/apt/lists/*
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+# install Python Flink
+RUN pip3 install apache-flink
```

Comment:
We should install a specific version; this will always fetch the latest.

Reply:
Yes， executing the command will install the latest version of PyFlink, maybe `RUN pip3 install apache-flink[==<SPECIFIC_VERSION>]` is better that the `[==<SPECIFIC_VERSION>]` is optional.

Reply:
I would suggest to update this as part of reworking the `docker.md` page as my change's intention is to only maintain the current information we have.

Reply:
Thanks a lot for the ping. I'll update the instructions.